tests/testthat/test-Data-methods.R

```diff
@@ -192,5 +208,31 @@ test_that("Update of DataDual works as expected", {
   object@cohort <- c(object@cohort, 4L, 4L)
 
   expect_valid(result, "DataDual")
-  expect_equal(result, object)
+  expect_identical(result, object)
+})
+
+# update-DataDA ----
+
```

Comment:
if we really need things like missing `trialtime` please make sure to test these edge cases enough, feels pretty risky in terms of code to me

Reply:
I think that it should be `assert_numeric(trialtime, lower = max(object@t0), len = min(length(y), 1))`

Reply:
```
assert_numeric(trialtime, lower = max(object@t0), len = min(length(y), 1))
```